CHANGELOG.md

```diff
@@ -15,12 +15,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - chore: upgrade sumologic terraform provider to 2.18 [#2399]
 - chore: update kubernetes-tools to 2.12.0 [#2472]
-- feat(otelcol/metrics): adjust metric otelcol configuration [#2474]
+- feat(otelcol/metrics): adjust metric otelcol configuration [#2474], [#2479]
```

Comment:
This says metrics, but you're changing configuration for logs.

Reply:
Yes, right, good catch